internal/resources/ekscluster/resource_ekscluster.go

```diff
@@ -85,6 +81,9 @@ var clusterSchema = map[string]*schema.Schema{
 		Description: "Wait timeout duration until cluster resource reaches READY state. Accepted timeout duration values like 5s, 45m, or 3h, higher than zero",
 		Default:     "default",
 		Optional:    true,
+		DiffSuppressFunc: func(k, oldValue, newValue string, d *schema.ResourceData) bool {
```

Comment:
This will be returning true always.. So do we need this condition still?

Reply:
I read it wrongly as the function returns false always.. I get the use case basically we want to suppress any diff for wait timeout.